tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts

```diff
@@ -99,18 +99,18 @@ async function automationGenerateInPipeline(
                     apiVersion: apiVersion,
                     sdkReleaseType: sdkReleaseType,
                     runMode: runMode as RunMode,
-                };                
+                };
                 const packageResult = await generateAzureSDKPackage(options);
                 outputJson.packages = [packageResult];
                 break;
             }
             default:
                 break;
         }
-
+       
         await codeOwnersAndIgnoreLinkGenerator({
```

Comment:
you can move to modular's `generateAzureSDKPackage` since we only support modular for now.